CHANGES.md

```diff
@@ -1,5 +1,19 @@
 # Changes
 
+## edge-20.7.4
+
+This edge release adds support for the new Kubernetes [EndpointSlice] resource
```

Comment:
Looks like this link needs a reference. Let's add

```markdown
[EndpointSlice](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/)
```
to the end of the changelog entry.

I tried to include the link reference in my suggested changes https://github.com/linkerd/linkerd2/pull/4785#discussion_r459646336 but it looks like GitHub screwed it up, because I also used a fenced code block _within_ my suggested code, so it ended the suggestion block too early. :/